### PR TITLE
First pass at new logfile infrastructure

### PR DESCRIPTION
**Pull request by lisazacarias**

### CODE REVIEW

File: quench_linac.py [hunk 1]
```diff
@@ -127,4 +133,20 @@ def validate_quench(self, wait_for_update: bool = False):
...
 
-QUENCH_CRYOMODULES = scLinac.CryoDict(cavityClass=QuenchCavity)
+class QuenchCryomodule(scLinac.Cryomodule):
+    def __init__(self, cryo_name, linac_object, cavity_class=QuenchCavity,
+                 magnet_class=scLinac.Magnet, rack_class=scLinac.Rack,
+                 is_harmonic_linearizer=False, ssa_class=scLinac.SSA,
+                 stepper_class=scLinac.StepperTuner, piezo_class=scLinac.Piezo):
+        super().__init__(cryo_name, linac_object, cavity_class=QuenchCavity)
+        self.logger = logging.getLogger(f'{self} quench resetter')
+        self.logger.setLevel(logging.DEBUG)
```
Comment (s-aderhold) [inline quench_linac.py:170]: Does each log entry have a timestamp? That would be helpful

Comment (lisazacarias) [reply quench_linac.py:170]: No! Good point

**Comment by lisazacarias** [comment]
Validate quench has been successfully tested in production.

File: quench_resetter.py [hunk 2]
```diff
@@ -1,53 +1,44 @@
...
 from lcls_tools.common.controls.pyepics.utils import PVInvalidError
-from lcls_tools.superconducting.sc_linac import Cryomodule
 from lcls_tools.superconducting.sc_linac_utils import (
     ALL_CRYOMODULES,
-    CavityFaultError,
     HW_MODE_ONLINE_VALUE,
+    CavityFaultError,
 )
-from quench_linac import QUENCH_MACHINE
+from quench_linac import QUENCH_MACHINE, QuenchCryomodule
```
Comment (nneveu) [inline quench_resetter.py:12]: Is this class only called when a cavity queches? Not a huge deal, but maybe rename to QuenchedCryomodule/QuenchedCavity?

**Review by nneveu — APPROVED** [review]

File: quench_resetter.py (hunk 2 above)
Comment (lisazacarias) [reply quench_resetter.py:12]: No it's just the name of the cryomodule class used for this application

**Pull request merged**